apps/aehttpserver/lib/aehttpserver/web/controllers/block_controller.ex

```diff
@@ -77,8 +75,7 @@ defmodule Aehttpserver.Web.BlockController do
           BlockValidation.block_header_hash(Block.genesis_block().header)
 
         hash ->
-          hash_bin = Bits.bech32_decode(hash)
-          hash_bin
+          Bits.decode58c(hash)
```

Comment:
Use the `Block.base58_decode` function to decode block hashes

Reply:
There is no such function